src/user/index.js

```diff
@@ -0,0 +1,50 @@
+import { computeAddress } from '@ethersproject/transactions'
```

Comment:
nice, these things were copy-pasted here and there (maybe still are? I think Session.js and Signer.js), good to get them in one place.